README.md

```diff
@@ -1,4 +1,4 @@
-# SYCL for Vitis 2020.2: experimental melting pot with Intel oneAPI DPC++ SYCL and triSYCL for Xilinx FPGA
+# SYCL for Vitis 2021.1: experimental melting pot with Intel oneAPI DPC++ SYCL and triSYCL for Xilinx FPGA
```

Comment:
Curious.
Why does this change appear here while it is already in the target branch?